Jellyfin.Api/Controllers/EnvironmentController.cs

```diff
@@ -47,7 +47,7 @@ public EnvironmentController(IFileSystem fileSystem, ILogger<EnvironmentControll
         [HttpGet("DirectoryContents")]
         [ProducesResponseType(StatusCodes.Status200OK)]
         public IEnumerable<FileSystemEntryInfo> GetDirectoryContents(
-            [FromQuery, BindRequired] string path,
+            [FromQuery, Required] string path,
```

Comment:
I think string has to be nullable for `Required` to work but not sure when it's a function arg.

Reply:
Just tested, `Required` works for both nullable and non-nullable strings the same